safe-global/web-core
# feat: relay batch modal

## What it solves

Resolves #1818 

## How this PR fixes it
Implements relaying for batch executions

## How to test it
1. In Goeli, fully sign 2+ transactions so that you can batch them
2. The "How to pay" info should be present in the Modal
3. Click "Send" will relay the Multisend transaction

## Screenshots
![Screenshot 2023-03-29 at 20 08 14](https://user-images.githubusercontent.com/32431609/228629157-7ebba1c4-f971-4366-86b9-17e39160935d.png)

## Checklist
